infrastructure/src/main/java/org/corfudb/infrastructure/LayoutServer.java

```diff
@@ -347,19 +362,22 @@ public synchronized void handleMessageLayoutCommit(
             return;
         }
 
-        Layout commitLayout = msg.getPayload().getLayout();
         if (!isBootstrapped(msg, ctx, r)) {
             return;
         }
-        long serverEpoch = getServerEpoch();
-        if (msg.getPayload().getEpoch() < serverEpoch) {
+
+        final long msgEpoch = msg.getPayload().getEpoch();
+        final long serverEpoch = getServerEpoch();
+        final Layout msgLayout = msg.getPayload().getLayout();
```

Comment:
I think the original name `commitLayout` is more descriptive, could you change it back?

Reply:
I changed it because it was ambiguous to begin with -- commitLayout implies that the layout has been committed, which is not the case. The other alternative is to use layoutToBeCommited, but the name was too long.